src/PageLayout/useStickyPaneHeight.ts

```diff
@@ -28,16 +28,16 @@ export function useStickyPaneHeight() {
 
     const topRect = contentTopEntry?.target.getBoundingClientRect()
     const bottomRect = contentBottomEntry?.target.getBoundingClientRect()
-
-    const stickyTopWithUnits = typeof stickyTop === 'number' ? `${stickyTop}px` : stickyTop
+    // custom sticky header's height with units
```

Comment:
I added this comment because there are 2 variables now here with a very similar name. `topOffset` and `offsetTop`. I think it is clear when reading the code but just to be double clear. Happy to update or remove if it is not necessary.

Reply:
😅 You're right, "offsetTop" is kind of a confusing name in this context. This is making me favor @siddharthkp's original suggestion of `offsetHeader`. What do you think? I'll let you make the final decision.

I'm going to push this to the next release to give you time to decide

Reply:
I think we can go with `offsetHeader` I think we wrote the docs good enough to explain what this prop does